<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWidget</class>
 <widget class="QWidget" name="mainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>481</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>669</width>
    <height>481</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>669</width>
    <height>481</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyLove</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/image/Love.ICO</normaloff>:/image/Love.ICO</iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>50</y>
     <width>241</width>
     <height>281</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>-3</x>
      <y>-2</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="QiXiHappylabel">
       <property name="minimumSize">
        <size>
         <width>134</width>
         <height>37</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>134</width>
         <height>37</height>
        </size>
       </property>
       <property name="text">
        <string>QiXiHappy</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
   <widget class="QToolButton" name="clickBtn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>200</width>
      <height>200</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="text">
     <string>clickBtn</string>
    </property>
    <property name="icon">
     <iconset resource="resource.qrc">
      <normaloff>:/image/clickBtn.png</normaloff>:/image/clickBtn.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>200</width>
      <height>200</height>
     </size>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QToolButton" name="ExitBtn">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>430</y>
     <width>40</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>ExitBtn</string>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/ExitBtn.png</normaloff>:/image/ExitBtn.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
   <property name="autoRaise">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="HeadLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="text">
    <string>Head_pic</string>
   </property>
  </widget>
  <action name="actionShow">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/OpenAction.png</normaloff>:/image/OpenAction.png</iconset>
   </property>
   <property name="text">
    <string>打开程序</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/image/CloseAction.png</normaloff>:/image/CloseAction.png</iconset>
   </property>
   <property name="text">
    <string>关闭程序</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
